react/src/fix_antd.css

```diff
@@ -39,6 +38,40 @@
   animation: loadingCircle 1s infinite linear;
 }
 
+.ant-tabs {
```

Comment:
`fix_antd.css` is for fixing this [issue](https://github.com/ant-design/ant-design-icons/issues/568) temporary. After this issue is resolved, this file will be removed.

Please do not add CSS for customization.

Reply:
😲 Oops, okay I'll fix it.